Tests/feaLib/builder_test.py

```diff
@@ -141,13 +142,20 @@ def check_feature_file(self, name):
                 font[tag].compile(font)
 
     def check_fea2fea_file(self, name, base=None, parser=Parser):
+        # TODO: Do better matching than this. Whitespace is mostly irrelevant
+        # except if it's inside a string. Name ID numbers can be hex, but are
+        # converted to ints so the diff fails. Name strings are allowed to be
+        # escaped in the source, but they become unescaped and diff fails.
         font = makeTTFont()
         fname = (name + ".fea") if '.' not in name else name
         p = parser(self.getpath(fname), glyphNames=font.getGlyphOrder())
         doc = p.parse()
         actual = self.normal_fea(doc.asFea().split("\n"))
+        actual = " ".join(actual).replace("\n", "").split(";")
```

Comment:
what are you trying to do here? Why split at semicolons?
Doesn't the `normal_fea` method already return list of lines stripped of the ending newline character?

Reply:
Like I put in the TODO above this way of comparing files has all kinds of problems so the comparison only works for some things and assumes all lines are exactly the same. Splitting on semicolons makes more sense than newlines because semicolons have meaning, but newlines can be arbitrary. Also, spaces, including newlines, are only relevant within a string so to truly compare fea we should also ignore all spaces except in strings. That's why I only did this minor adjustment and added a TODO. This can be much better, but isn't necessary right now.

Reply:
> Splitting on semicolons makes more sense than newlines because semicolons have meaning, but newlines can be arbitrary.

but the arbitrary newlines are the ones that feaLib asFea() itself puts, so at least they should be consistent and predictable. 
Anyway I'm not convinced this is necessary for the present PR. I would prefer if this was addressed separately if at all.

Reply:
Ok, fine to drop it from this PR.